src/Proxies/GoogleGtagJs.php

```diff
@@ -53,7 +53,7 @@ public function is_adding_framework() {
 	 */
 	private function is_gtag_page(): bool {
 		$standard_tracking_enabled = 'yes' === $this->wcga_settings['ga_standard_tracking_enabled'];
-		$is_wc_page                = is_order_received_page() || is_woocommerce() || is_cart() || is_checkout();
+		$is_wc_page                = is_order_received_page() || is_woocommerce() || is_cart() || is_checkout() || is_product();
```

Comment:
The function `is_woocommerce` is defined as returning true for `is_shop() || is_product_taxonomy() || is_product()` so it's not necessary to add the `is_product` check again.

Reply:
Thanks for your suggestions. Removed.